cmd/delete-oldsnapshots/delete-oldsnapshots.go

```diff
@@ -0,0 +1,62 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/aws/aws-sdk-go/service/ec2/ec2iface"
+)
+
+var (
+	awsRegion = flag.String("aws-region", "eu-west-2", "AWS_REGION")
+	ownerId   = flag.String("owner-id", "754256621582", "OWNER_ID")
+	daysOld   = flag.Int("days-old", 400, "DAYS_OLD")
+)
+
+// Delete snapshots older than given days
+func deleteOldSnapshots(svc ec2iface.EC2API, days int) error {
+	output, err := svc.DescribeSnapshots(&ec2.DescribeSnapshotsInput{Filters: []*ec2.Filter{
+		{
+			Name: aws.String("owner-id"),
+			Values: []*string{
+				aws.String(*ownerId),
+			},
+		},
+	}})
+	if err != nil {
+		log.Fatal("Error to describe snapshots:", err.Error())
```

Comment:
Instead of logging here is it worth returning the error and logging from the caller?